index.js

```diff
@@ -39,7 +40,12 @@ async function run() {
         try {
             await exec.exec(command);
         } catch (err) {
-            core.setFailed('The ZAP Baseline scan has failed, starting to analyze the alerts. err: ' + err.toString());
+            if (failAction == 'true') {
```

Comment:
We should fail the action always when the exit code is 3, that means there's a problem with the script/scan itself (e.g. scan not properly configured or an error while running the script). In that case there's probably no point processing the report (there shouldn't be any).

Reply:
@thc202 is there a way to simulate the code 3 behavior for testing?

Reply:
The simplest is to pass a target that's not `http` or `https`.

Reply:
Fixed